configs/frameworks/buildings_nzc.yaml

```diff
@@ -100,21 +125,38 @@ nodes:
   output_dimensions: [heating_type, ghg, scope]
   input_nodes:
   - id: building_energy_use
-    to_dimensions:
-    - id: heating_type
     from_dimensions:
     - id: building_energy_class
       flatten: true
+    to_dimensions:
+    - id: heating_type
+    - id: scope
   - id: heating_emission_factor
+    from_dimensions:
+    - id: pollutant
+      categories: [co2]
+      flatten: true
     to_dimensions:
     - id: heating_type
-      categories: [local_heating]
     - id: ghg
       categories: [co2e]
-    - id: scope
-      categories: [scope2]
   output_nodes:
   - id: building_emissions
     from_dimensions:
     - id: heating_type
       flatten: true
+
+#actions:
+
+#- id: renovation_rate
+#  name: Renovation rate
+#  description: 'Default 1 %, BPIE (2011) - "Europe's buildings under the microscope". http://bpie.eu/wp-content/uploads/2015/10/HR_EU_B_under_microscope_study.pdf'
+#  type: shift.ShiftAction
+#  quantity: fraction
+#  unit: '%'
+#  params:
+#  - id: shift
+#    unit: '%'
+#    values:
+#    - year: 2023
+#      value: 1
```

Comment:
There are some of these sections that are commented out (another one in https://github.com/kausaltech/kausal-paths/pull/42/commits/63edc25b8f21b99ecb18a995c5659a6c129dffa2#diff-599cb6921e50bc26beac4d823062c6b1c923ae43a3a1c6277158c6b3138ed27cR289 ) , are they left in intentionally?

In general throughout Paths code there are some entire sections of code that are commented out. It can be a bit confusing and difficult to know if there is a purpose behind leaving those sections in or is it just something that should be thrown away, actually I just had a case where I spent an hour or so investigating a piece of code like that until I finally concluded that the code has been rewritten elsewhere and can be deleted here (whereas for the original developer it would have taken just some seconds to delete the commented code). So I think unless there is a solid intention to use the code later it would be better to leave any commented code out to begin with.

Reply:
Agreed. The practical problem for me is that usually these are blocks that I intend to use later but then I don't, and afterwards I don't remember that they are sitting there. I'll try to improve and scan through yamls that I am working on, deleting old stuff.

Reply:
Great :+1: